### PR TITLE
Feature/alcs 767

### PR DESCRIPTION
**Pull request by mhuseinov**
- new reconsideration fields for applications originated in portal
- always display review outcome date in edit window for reconsideration of type 33

### CODE REVIEW

File: alcs-frontend/src/app/features/application/post-decision/edit-reconsideration-dialog/edit-reconsideration-dialog.component.html [hunk 1]
```diff
@@ -37,6 +37,42 @@ <h2 class="card-title">Edit Reconsideration</h2>
...
+          id="reviewOutcome"
+          formControlName="type"
+          name="reviewOutcome"
+          (change)="onTypeReconsiderationChange($event.value)"
+        >
+          <mat-button-toggle *ngFor="let type of codes" value="{{ type.code }}">{{ type.label }}</mat-button-toggle>
+        </mat-button-toggle-group>
+      </div>
+
+      <div *ngIf="isOriginatedInPortal">
```
Comment (dhaselhan) [inline alcs-frontend/src/app/features/application/post-decision/edit-reconsideration-dialog/edit-reconsideration-dialog.component.html:52]: I believe the benefit of doing this after the "hard fork" is that you don't need to make these fields conditional.

File: alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-input-v2.component.ts [hunk 2]
```diff
@@ -238,7 +238,7 @@ export class DecisionInputV2Component implements OnInit, OnDestroy {
       .filter(
         (modification) =>
           (existingDecision && existingDecision.modifies?.uuid === modification.uuid) ||
-          (modification.reviewOutcome.code === 'PRC' && modification.resultingDecision === null)
```
Comment (dhaselhan) [inline alcs-frontend/src/app/features/application/decision/decision-v2/decision-input/decision-input-v2.component.ts:241]: Why is this being changed?

File: alcs-frontend/src/app/features/application/post-decision/edit-reconsideration-dialog/edit-reconsideration-dialog.component.html (hunk 1 above)
Comment (mhuseinov) [reply alcs-frontend/src/app/features/application/post-decision/edit-reconsideration-dialog/edit-reconsideration-dialog.component.html:52]: The ticket requested to have the reconsideration to be applied only to the once that originated in the portal. Do we ignore those apps that were created before the etl?

Comment (mhuseinov) [reply alcs-frontend/src/app/features/application/post-decision/edit-reconsideration-dialog/edit-reconsideration-dialog.component.html]: Updated ticket according to requirements change. The OriginatedInPortal is not relevant anymore and was removed

**Review by dhaselhan — APPROVED** [review]
LGTM 👍

**Pull request merged**